src/client/app/components/about.ts

```diff
@@ -65,13 +100,34 @@ export class About implements AfterViewInit {
         this.showChange.emit(false);
     }
 
-    cache = `
-    ${Strings().aboutStorage}
-    ${storageSize(localStorage, `playground_${environment.current.host}_snippets`, Strings().aboutSnippets)}
-    ${storageSize(sessionStorage, 'playground_intellisense', Strings().aboutIntellisense)}
-    `;
-
     setLanguage(languageCode: string) {
         this.currentChosenLanguage = languageCode;
     }
+
+    async changeConfig(newConfig: string) {
+        if (config[newConfig] && config[newConfig].name !== environment.current.config.name) {
+            let currentConfigName = environment.current.config.name.toLowerCase();
+            let result = await this._effects.alert(
+                this.strings.changeEnvironmentConfirm,
+                `${this.strings.aboutSwitchEnvironment} ${this.configs.find(c => c.name === currentConfigName).value } to ${this.configs.find(c => c.name === this.selectedConfig).value }`,
+                this.strings.okButtonLabel,
+                this.strings.cancelButtonLabel
+            );
+
+            if (result === this.strings.cancelButtonLabel) {
+                this.selectedConfig = this.configs.find(c => c.name === currentConfigName).name;
+                this._changeDetector.detectChanges();
+                return;
+            }
+
+            let originEnvironment = window.localStorage.getItem('originEnvironment');
```

Comment:
Talked about this offline. Since we're using the set `editorUrl` from our defined `config` variable, adding a query string parameter in this manner should be fine.